lib/compat/wordpress-6.3/class-gutenberg-rest-block-patterns-controller-6-3.php

```diff
@@ -158,4 +158,33 @@ public function get_item_schema() {
 
 		return $this->add_additional_fields_schema( $this->schema );
 	}
+
+	/**
+	 * Retrieves all block patterns.
+	 *
+	 * @since 6.0.0
+	 * @since 6.2.0 Added migration for old core pattern categories to the new ones.
+	 *
+	 * @param WP_REST_Request $request Full details about the request.
+	 * @return WP_REST_Response|WP_Error Response object on success, or WP_Error object on failure.
+	 */
+	public function get_items( $request ) {
+		if ( ! $this->remote_patterns_loaded ) {
+			// Load block patterns from w.org.
+			gutenberg_load_remote_block_patterns(); // Patterns with the `core` keyword.
+			gutenberg_load_remote_featured_patterns(); // Patterns in the `featured` category.
+			gutenberg_register_remote_theme_patterns(); // Patterns requested by current theme.
+
```

Comment:
I moved the `get_items()` method from `Gutenberg_REST_Block_Patterns_Controller_6_2` to `Gutenberg_REST_Block_Patterns_Controller_6_3` because the following methods were updated in 6.3:

- gutenberg_load_remote_block_patterns
- gutenberg_load_remote_featured_patterns
- gutenberg_register_remote_theme_patterns

The consequence is that Gutenberg users that are running 6.2 will get the benefit of these 6.3 changes.

Does that sound legit @aaronrobertshaw ?

Reply:
That's sounds right, yes.

Reply:
Thanks for checking 🙇🏻